### ISSUE TITLE
Error on trying to open stencils or create new form

### ISSUE THREAD

**Issue by umkasanki** [verbb/formie#2073]
### Describe the bug

I have such error on trying to open stencils or create a new form after upgrade to Craft 5.

```
yii\base\UnknownPropertyException: Setting unknown property: verbb\formie\fields\Hidden::vid in /app/vendor/yiisoft/yii2/base/Component.php:217
Stack trace:
#0 /app/vendor/craftcms/cms/src/helpers/App.php(1479): yii\base\Component->__set('vid', 'new7215-7087')
#1 /app/vendor/craftcms/cms/src/base/Model.php(84): craft\helpers\App::configure(Object(verbb\formie\fields\Hidden), Array)
#2 /app/vendor/verbb/formie/src/base/Field.php(264): craft\base\Model->__construct(Array)
#3 /app/vendor/verbb/formie/src/fields/Hidden.php(62): verbb\formie\base\Field->__construct(Array)
#4 [internal function]: verbb\formie\fields\Hidden->__construct(Array)
#5 /app/vendor/yiisoft/yii2/di/Container.php(419): ReflectionClass->newInstanceArgs(Array)
#6 /app/vendor/yiisoft/yii2/di/Container.php(170): yii\di\Container->build('verbb\\formie\\fi...', Array, Array)
#7 /app/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get('verbb\\formie\\fi...', Array, Array)
#8 /app/vendor/craftcms/cms/src/Craft.php(70): yii\BaseYii::createObject(Array, Array)
#9 /app/vendor/craftcms/cms/src/helpers/Component.php(142): Craft::createObject(Array)
#10 /app/vendor/verbb/formie/src/services/Fields.php(433): craft\helpers\Component::createComponent(Array, 'verbb\\formie\\ba...')
#11 /app/vendor/verbb/formie/src/models/FieldLayoutRow.php(95): verbb\formie\services\Fields->createField(Array)
#12 /app/vendor/yiisoft/yii2/base/Component.php(180): verbb\formie\models\FieldLayoutRow->setFields(Array)
#13 /app/vendor/craftcms/cms/src/helpers/App.php(1479): yii\base\Component->__set('fields', Array)
#14 /app/vendor/craftcms/cms/src/base/Model.php(84): craft\helpers\App::configure(Object(verbb\formie\models\FieldLayoutRow), Array)
#15 /app/vendor/verbb/formie/src/models/FieldLayoutPage.php(127): craft\base\Model->__construct(Array)
#16 /app/vendor/yiisoft/yii2/base/Component.php(180): verbb\formie\models\FieldLayoutPage->setRows(Array)
#17 /app/vendor/craftcms/cms/src/helpers/App.php(1479): yii\base\Component->__set('rows', Array)
#18 /app/vendor/craftcms/cms/src/base/Model.php(84): craft\helpers\App::configure(Object(verbb\formie\models\FieldLayoutPage), Array)
#19 /app/vendor/verbb/formie/src/models/FieldLayoutPage.php(53): craft\base\Model->__construct(Array)
#20 /app/vendor/verbb/formie/src/models/StencilData.php(145): verbb\formie\models\FieldLayoutPage->__construct(Array)
#21 /app/vendor/verbb/formie/src/models/Stencil.php(65): verbb\formie\models\StencilData->__construct(Array)
#22 /app/vendor/verbb/formie/src/services/Stencils.php(62): verbb\formie\models\Stencil->__construct(Array)
#23 /app/vendor/verbb/formie/src/controllers/StencilsController.php(33): verbb\formie\services\Stencils->getAllStencils()
#24 [internal function]: verbb\formie\controllers\StencilsController->actionIndex()
#25 /app/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#26 /app/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#27 /app/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('index', Array)
#28 /app/vendor/craftcms/cms/src/web/Application.php(350): yii\base\Module->runAction('formie/stencils...', Array)
#29 /app/vendor/yiisoft/yii2/web/Application.php(103): craft\web\Application->runAction('formie/stencils...', Array)
#30 /app/vendor/craftcms/cms/src/web/Application.php(318): yii\web\Application->handleRequest(Object(craft\web\Request))
#31 /app/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#32 /app/web/index.php(12): yii\base\Application->run()
#33 {main}
```
![image](https://github.com/user-attachments/assets/b8d38f8f-4da1-4521-9782-32d3f9100e7d)
![image](https://github.com/user-attachments/assets/dc2f162c-8093-434c-84a0-4da83ae42079)

### Steps to reproduce

1. Update Craft from 4 to 5
2. Open Formie / Settings / Stencils

### Form settings

- Multi-page form: Yes
- Submission Method: Ajax 
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

Craft Pro 5.4.5.1

### Plugin version

3.0.7

### Multi-site?

No

### Additional context

Configured Integration with HubSpot

**Comment by engram-design** [comment]
Sounds like this stencil wasn't properly upgraded from Formie 2. This wasn't something you've manually carried over from another (older) project, or you're using a base project that's out of date?

**Comment by umkasanki** [comment]
This is a new stencil I created for this project in Formie 2  a week before the update.
Is there any way to run the stencil upgrade?

**Comment by umkasanki** [comment]
I tried to run Craft update 3 times. Stencils are not updated for some reason..
I also tried rerunning the `m240318_000000_migrate_stencils` migration.
That didn't solve the problem. It looks like this migration isn't working..

**Comment by engram-design** [comment]
[This migration](https://github.com/verbb/formie/blob/craft-5/src/migrations/m240318_000000_migrate_stencils.php) should be updating your stencils, but I can just add some handling in the main stencil class to make things easier.

Before I do so, and to check if there's any other invalid properties, would you mind sending your stencil YAML through to `support@verbb.io` so I can take a look?

**Comment by umkasanki** [comment]
I've tried to run [this migration](https://github.com/verbb/formie/blob/craft-5/src/migrations/m240318_000000_migrate_stencils.php). That didn't solve the problem. 

> would you mind sending your stencil YAML through to support@verbb.io so I can take a look?

Done.

**Comment by engram-design** [comment]
Thanks! Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.7"`.

**Comment by engram-design** [comment]
Fixed in [3.0.8](https://github.com/verbb/formie/releases/tag/3.0.8)
